application/src/main/kotlin/net/corda/v5/application/flows/RPCRequestData.kt

```diff
@@ -0,0 +1,16 @@
+package net.corda.v5.application.flows
+
+/**
+ * A wrapper around the request data for RPC started flows.
+ *
+ * An RPC started flow will receive an instance of this interface, which can be used to retrieve the request body.
+ */
+interface RPCRequestData {
```

Comment:
Would it be worth just naming this `RPCRequest`? Only reason I say that, is that it's more generic so if we want to add new things to the interface for flows to access, we can do so without breaking apis, adding new ways to do things or having a badly named interface.

While on this subject, should we also pass in the client id into the flow, as part of this interface. It might be useful to users for logging things and currently, they would have to pass it in via the request body. Or, we could provide a way to access it like we do flow id? I know I've gone off on a tangent at this point.

Reply:
The `clientID` stuff I think we should look at separately. I don't want this PR to get too off topic and I think there's some overlap there with the context propagation work we need to think about.

I'd be careful about making this object more than the request body - there's a danger of having two mechanisms to do similar things with services the flow provides too. So functions for working with the request body are fine, but extra things there I'm less sure is a good idea.